ui/apps/dashboard/src/app/(organization-active)/(dashboard)/env/[environmentSlug]/apps/page.tsx

```diff
@@ -1,19 +1,13 @@
-import dynamic from 'next/dynamic';
 import { NewButton } from '@inngest/components/Button';
 import { Header } from '@inngest/components/Header/Header';
 import { Link } from '@inngest/components/Link/Link';
 import { Tooltip, TooltipContent, TooltipTrigger } from '@inngest/components/Tooltip/Tooltip';
 import { RiAddLine, RiQuestionLine } from '@remixicon/react';
 
 import { StatusMenu } from '@/components/Apps/StatusMenu';
-import { ServerFeatureFlag } from '@/components/FeatureFlags/ServerFeatureFlag';
 import { pathCreator } from '@/utils/urls';
 import { Apps } from './Apps';
 
-const NewUser = dynamic(() => import('@/components/Surveys/NewUser'), {
```

Comment:
Will we need the NewUser file in the future? Or should it go as well?

Reply:
I was thinking of turning it into a generic survey widget, but I can remove it.